scripts/sync_makefiles.sh

```diff
@@ -15,59 +15,85 @@ orgs="prometheus prometheus-community"
 
 GITHUB_TOKEN="${GITHUB_TOKEN:-}"
 if [ -z "${GITHUB_TOKEN}" ]; then
-	echo -e "\e[31mGitHub token (GITHUB_TOKEN) not set. Terminating.\e[0m"
-	exit 1
+  echo -e "\e[31mGitHub token (GITHUB_TOKEN) not set. Terminating.\e[0m"
+  exit 1
 fi
 
 # Go to the root of the repo
-cd "$(git rev-parse --show-cdup)"
+cd "$(git rev-parse --show-cdup)" || exit 1
 
 source_makefile="$(pwd)/Makefile.common"
 source_checksum="$(sha256sum Makefile.common | cut -d' ' -f1)"
 
-tmp_dir=$(mktemp -d)
-trap "rm -rf ${tmp_dir}" EXIT
+tmp_dir="$(mktemp -d)"
+trap 'rm -rf "${tmp_dir}"' EXIT
+
+fetch_repos() {
+  local url="https://api.github.com/users/${1}/repos?per_page=100"
+  curl --retry 5 --silent -u "${git_user}:${GITHUB_TOKEN}" "${url}" 2>/dev/null |
+    jq -r '.[] | select( .name != "prometheus" ) | .name'
+}
+
+push_branch() {
+  # stdout and stderr are redirected to /dev/null otherwise git-push could leak
+  # the token in the logs.
+  git push --quiet \
+    "https://${GITHUB_TOKEN}:@github.com/${1}" \
+    --set-upstream "${branch}" 1>/dev/null 2>&1
+}
+
+post_template='{"title":"%s","base":"master","head":"%s","body":"%s"}'
+post_json="$(printf "${post_template}" "${pr_title}" "${branch}" "${pr_msg}")"
+post_pull_request() {
+  curl --show-error --silent --fail \
+    -u "${git_user}:${GITHUB_TOKEN}" \
+    -X POST \
```

Comment:
I'd suggest we remove that line: https://daniel.haxx.se/blog/2015/09/11/unnecessary-use-of-curl-x/

Reply:
Hah, yes, that's fine with me.